lib/Echidna/Processor.hs

```diff
@@ -0,0 +1,133 @@
+{-# LANGUAGE LambdaCase #-}
+
+module Echidna.Processor where
+
```

Comment:
let's import `Data.Foldable  (toList)` and use the generic one instead of `M.toList` etc

Reply:
We can't do that. They don't do the same thing for mappings: `M.toList` returns `[(Text, Value)]` while `DF.toList` returns `[Value]`